README.rst

```diff
@@ -89,7 +89,7 @@ There are two date arithmetic traps in the Gregorian calendar:
 
 In both cases, the result must be changed to the first day of the next month.
 
-Provided periods are represented by [start date inclusive, end date exclusive),
+Provided periods are represented by (start date inclusive, end date exclusive),
```

Comment:
This is the correct and intended mathematical notation. English uses [0, 1) where French uses [0, 1[.